dbus/src/blocking.rs

```diff
@@ -204,20 +211,46 @@ impl $c {
         self.remove_match_no_cb(&mr.match_str())
     }
 
+    /// If true, configures the connection to send signal messages to all matching [`MatchRule`]
+    /// filters added with [`add_match`](Self::add_match) rather than just the first one. This will
+    /// result in the messages being duplicated, and so the message serial will be lost, but this is
+    /// generally not a problem for signals.
+    ///
+    /// This is false by default, for a newly-created connection.
+    pub fn all_signal_matches(&mut self, all_signal_matches: bool) {
```

Comment:
Should be named `set_all_signal_matches`

Reply:
Done.